### ISSUE TITLE
Unable to access IQ-TREE Google group

### ISSUE THREAD

**Issue by HS6986** [iqtree/iqtree2#369]
Dear IQ-TREE Developers,

I have been using IQ-TREE Google group to view past discussions without joining the group. However, for at least several months now, when [I try to access the group](https://groups.google.com/g/iqtree), I get a “Banned content warning” and cannot view the content.

![Screenshot 2024-12-08 161126](https://github.com/user-attachments/assets/c11934a2-81f3-4abb-a3a5-d9d16a38c5cc)

Also:
- Discussions in IQ-TREE Google group no longer seem to appear in Google search results.
- When I try to access URLs of discussions I have saved (e.g., [this page](https://groups.google.com/g/iqtree/c/j884eSJiugY)), it says “You don't have permission to access this content” and asks me to join the group, but I cannot with an error.

![Screenshot 2024-12-08 161207](https://github.com/user-attachments/assets/9ef42664-fc61-425a-b13f-973ab7b928be)
![Screenshot 2024-12-08 161342 - frame at 0m4s](https://github.com/user-attachments/assets/0ff35ed5-65a8-48d5-bf90-4bc6c29f193a)

- When I try to join the group from [this page](https://groups.google.com/g/iqtree/about), I cannot with the same error as above.

These behaviors are confirmed in Google Chrome, Microsoft Edge, and Mozilla Firefox, at least in my environment. If these issues are widespread, they may be significantly affecting the accessibility of valuable information for IQ-TREE users.

I would appreciate it if you could check these issues and fix them if necessary.

**Comment by ctaylor235** [comment]
Yes, I am seeing the same in Safari. Currently a lot of links on the main website point to the Google group for further help, bug submission etc. so none of them are available.

**Comment by michelleleger** [comment]
Looks like the group's been banned for "spam, malware or other malicious content" o_0

**Comment by HS6986** [comment]
Dear IQ-TREE Developers,

I understand that you are very busy, but I would greatly appreciate it if you could address this issue when you are available. I consider this issue a very serious matter.

**Comment by evolbeginner** [comment]
> Dear IQ-TREE Developers,
> 
> I understand that you are very busy, but I would greatly appreciate it if you could address this issue when you are available. I consider this issue a very serious matter.

Hi friend,

I am not on iqtree development team but I think iqtree's google grp is closed and it is moving to github for any Qs and bug reports.

best,
sishuo

**Comment by roblanf** [comment]
Spot on @evolbeginner. 

@HS6986 if you have questions, please ask them on the discussion forum.

Rob

**Comment by HS6986** [comment]
Thank you very much for your response.

I think it is a loss for us not to have access to the important information discussed in the IQ-TREE Google group in the past, but I understand that the group has been closed and I will follow the new approach on GitHub.

I appreciate your time and support.
